Lib/netrc.py

```diff
@@ -2,7 +2,8 @@
 
 # Module and documentation by Eric S. Raymond, 21 Dec 1998
 
-import os, stat
+import os
+import stat
```

Comment:
Let's undo formatting to make sure only relevant changes are visible in the diff.
```suggestion
import os, stat
```